docs/manual/templates.rst

```diff
@@ -119,11 +119,32 @@ in the template. Typically, these vary per form!
 
 These are additional variables and statements available to the template.
 
-===================  ===========================================================================
-Element              Description
-===================  ===========================================================================
-``{% summary %}``    A full summary of all elements marked to show in the email.
-===================  ===========================================================================
+===================================  ===========================================================================
+Element                              Description
+===================================  ===========================================================================
+``{% summary %}``                    A full summary of all elements marked to show in the email.
+``{% appointment_information %}``    The information about the appointment to show in the email.
+``{% get_appointment_links %}``      Retrieves relevant links about the appointment.
+===================================  ===========================================================================
+
+**get_appointment_links example**
+
+.. tabs::
+
+   .. tab:: Template
+
+      .. code:: django
+
+         {% get_appointment_links as links %}
+         Cancel Appointment: {{ links.cancel_url|urlize }}
+
+
+   .. tab:: Rendered
+
+      .. code:: text
+
+         Cancel Appointment: http://fake.nl/api/v1/submission-uuid/token/verify/
+
```

Comment:
Is this okay for an example?

![Screenshot 2021-09-14 at 15 51 19](https://user-images.githubusercontent.com/60747362/133270155-2caf68b0-f2c5-46e5-bb42-70231072d27d.png)

![Screenshot 2021-09-14 at 15 51 25](https://user-images.githubusercontent.com/60747362/133270153-a8ec1c84-6b8a-49b5-b1c5-ee734bf8f0ee.png)

Reply:
Yes, I like